ledger/src/blockstore.rs

```diff
@@ -6070,40 +6069,16 @@ pub mod tests {
         blockstore.insert_shreds(shreds, None, false).unwrap();
         assert!(recvr.recv_timeout(timer).is_err());
 
-        // Insert a shred for each slot that doesn't make a consecutive block, we
```

Comment:
not sure what this is intended to check that other checks do not already check

Reply:
I am not following the logic here... why does it then do all the magic with indices?

Reply:
I'm guessing this is just one of many ways we could have made sure these shreds are far in the future (from an index perspective) compared to what was already inserted. But I can't say for sure what the original intent was

Reply:
Same here. Hope Steve or Zach know more.